build/version.go

```diff
@@ -4,11 +4,13 @@ var CurrentCommit string
 var BuildType int
 
 const (
-	BuildDefault  = 0
-	BuildMainnet  = 0x1
-	Build2k       = 0x2
-	BuildDebug    = 0x3
-	BuildCalibnet = 0x4
```

Comment:
I'd keep this as explicit hex values - we're doing some bit-hacky things with these (in 2k/debug builds)

Reply:
hm, okay. in either case, they are type int, I think it's equivilent to use iota in this case. Will change back to explicit hex anyway.